stigs/system/ksp-block-stig-rhel-v-230534-1.yaml

```diff
@@ -0,0 +1,22 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+# https://www.stigviewer.com/stig/red_hat_enterprise_linux_8/2021-12-03/finding/V-230534
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-block-stig-rhel-v-230534-1
+  namespace: default # Change your namespace
+spec:
+  tags: ["STIG","RHEL"]
+  message: "Alert! adding new users access has been blocked"
```

Comment:
@yasin-cs-ko-ak Can you give more suitable message here

Reply:
@salman-accuknox I couldn't think of it anything right now. Could you suggest one?

Reply:
access to passwd file is blocked